torch/testing/_internal/common_utils.py

```diff
@@ -823,6 +824,33 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
         r = func(*args, **kwargs)
         return r
 
+# Run PyTorch tests with TorchDynamo
+# TODO - Remove this in next PR to have easy revert strategy in case of unstable
+# CI
+# TEST_WITH_TORCHDYNAMO = os.getenv('PYTORCH_TEST_WITH_DYNAMO') == '1'
```

Comment:
Will uncomment this in next PR, so that the revert is not so bad in the presence of unstable CI.